executor/analyze_col.go

```diff
@@ -107,11 +108,15 @@ func (e *AnalyzeColumnsExec) buildResp(ranges []*ranger.Range) (distsql.SelectRe
 	var builder distsql.RequestBuilder
 	reqBuilder := builder.SetHandleRangesForTables(e.ctx.GetSessionVars().StmtCtx, []int64{e.TableID.GetStatisticsID()}, e.handleCols != nil && !e.handleCols.IsInt(), ranges, nil)
 	builder.SetResourceGroupTagger(e.ctx.GetSessionVars().StmtCtx.GetResourceGroupTagger())
+	startTS := uint64(math.MaxUint64)
```

Comment:
I tend to change startTS like following in order to reduce `getSnapshotTS`

https://github.com/pingcap/tidb/blob/11625d46af2612d6482551394f95e0eb16def771/executor/builder.go#L2161-L2169

Reply:
We still need snapshotTS when dealing with concurrent analyzes in the following:
https://github.com/pingcap/tidb/blob/51b8884feaaedbde1c363e39fa5d47c33c52c960/statistics/handle/handle.go#L1147-L1155